v2/awsv1shim/session.go

```diff
@@ -145,6 +145,13 @@ func GetSession(ctx context.Context, awsC *awsv2.Config, c *awsbase.Config) (*se
 			})
 			r.Retryable = aws.Bool(false)
 		}
+
+		if r.IsErrorExpired() {
```

Comment:
There should be a corresponding update to `networkErrorShortcutter.RetryDelay` for calls using the AWS SDK for Go v2

Reply:
Not needed since V2 won't retry 403 Forbidden with standard retryer ([not retryable and not in the list of errors to be retried](https://github.com/aws/aws-sdk-go-v2/blob/4616baebcf44f8b1eed7cee18ae71b6ad51db99a/aws/retry/standard.go#L79-92)).